internal/config/config.go

```diff
@@ -14,15 +15,9 @@ type Listener struct {
 	AnonSSH        string `toml:"anon_ssh"`
 }
 
-type Module struct {
-	Name string   `toml:"name"`
-	Path string   `toml:"path"`
-	ACL  []string `toml:"acl"`
-}
-
 type Config struct {
-	Listeners []Listener `toml:"listener"`
-	Modules   []Module   `toml:"module"`
+	Listeners []Listener      `toml:"listener"`
+	Modules   []rsyncd.Module `toml:"module"`
```

Comment:
As per our discussion, no need to expose this config package if `Module` is defined by `rsyncd` package itself.

Also related: we discussed earlier renaming config -> rsyncdconfig. I reverted the rename and left it out from this PR, since the package is not needed as public.

If you want, I can send another PR renaming `internal/config` -> `internal/rsyncdconfig`. But it wasn't necessary here.

Reply:
If you have time for the `internal/config` → `internal/rsyncdconfig` change as well, I’ll gladly take that as a separate pull request later :)